native/src/main/native/org/mozilla/jss/crypto/Algorithm.c

```diff
@@ -124,7 +124,7 @@ JSS_AlgInfo JSS_AlgTable[NUM_ALGS] = {
 /* 35 */    {CKM_AES_CBC_PAD, PK11_MECH},
 /* 36 */    {CKM_RC2_CBC_PAD, PK11_MECH},
 /* 37 */    {CKM_RC2_KEY_GEN, PK11_MECH},
-/* 38 */    {SEC_OID_SHA256, SEC_OID_TAG},
+/* 38 */    {SEC_OID_SHA256, PK11_MECH},
 /* 39 */    {SEC_OID_SHA384, SEC_OID_TAG},
 /* 40 */    {SEC_OID_SHA512, SEC_OID_TAG},
 /* 41 */    {SEC_OID_PKCS1_SHA256_WITH_RSA_ENCRYPTION, SEC_OID_TAG},
```

Comment:
I take it we will need entries here for 384 and 512?

Reply:
Yes